src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -582,13 +582,49 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
       def processAccountsPage(
           taggedAccounts: List[BlockTagged[AccountsIndex]],
           taggedDelegateKeys: List[BlockTagged[DelegateKeys]]
-      ): Future[(Option[Int], Option[Int], List[BlockTagged[DelegateKeys]])] =
-        db.run(TezosDb.writeAccountsAndCheckpointDelegates(taggedAccounts, taggedDelegateKeys))
-          .map {
+      ): Future[(Option[Int], Option[Int], List[BlockTagged[DelegateKeys]])] = {
+        for {
+          activatedOperations <- fetchActivatedOperationsByLevel(taggedAccounts.map(_.blockLevel).distinct)
+          activatedAccounts <- db.run(TezosDb.findActivatedAccountIds)
```

Comment:
This might become very expensive if the number of accounts grows a lot. Can we put in an index for now?

Reply:
sure, added